lib/ansible/module_utils/vmware.py

```diff
@@ -1093,6 +1093,35 @@ def get_all_host_objs(self, cluster_name=None, esxi_host_name=None):
 
         return host_obj_list
 
+    def host_version_at_least(self, version=None, vm_obj=None, host_name=None):
+        """
+        Check that the ESXi Host is at least a specific version number
+        Args:
+            vm_obj: virtual machine object, required one of vm_obj, host_name
+            host_name (string): ESXi host name
+            version (tuple): a version tuple, for example (6, 7, 0)
+        Returns: bool
+        """
+        if vm_obj:
+            host_system = vm_obj.summary.runtime.host
+        elif host_name:
+            host_system = self.find_hostsystem_by_name(host_name=host_name)
+        else:
+            self.module.fail_json(msg='VM object or ESXi host name must be set one.')
+        if host_system and version:
+            host_version = host_system.summary.config.product.version
+            if int(host_version.split('.')[0]) < version[0]:
```

Comment:
You could replace this entire block with:
```
            from distutils.version import StrictVersion
            return StrictVersion(host_version) >= StrictVersion('.'.join(map(str,version)))
```

Reply:
@Tomorrow9 Are you including these changes ?